### PR TITLE
Output strings instead of indices in physical device feature error messages

### LINKED ISSUE #1924: Improve output of VkPhysicalDeviceFeature struct error
With the new device simulation layer, it will be common to get errors like this:

While calling vkCreateDevice(), requesting feature #21 in VkPhysicalDeviceFeatures struct, which is not available on this device.

This is not as helpful as a message as it could be.  Basically this is a feature request to do what is in the TODO line here:

https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/blob/master/layers/core_validation.cpp#L2004

### PR DESCRIPTION
**Pull request by mark-lunarg**
Fixes #1924.  Tacked a generated helper function onto the enum string helper header.

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -2012,9 +2012,9 @@ static bool ValidateRequestedFeatures(instance_layer_data *instance_data, const
             // TODO: Add index to struct member name helper to be able to include a feature name
```
Comment (mikes-lunarg) [inline layers/core_validation.cpp]: TODONE

File: scripts/helper_file_generator.py [hunk 2]
```diff
@@ -364,6 +364,24 @@ def GenerateEnumStringConversion(self, groupName, value_list):
...
     #
+    # Tack on a helper which, given an index into a VkPhysicalDeviceFeatures structure, will print the corresponding feature name
+    def DeIndexPhysDevFeatures(self):
+        pdev_members = None
+        for name, members, ifdef in self.structMembers:
+            if name == 'VkPhysicalDeviceFeatures':
+                pdev_members = members
+                break
+        deindex = '\n'
+        deindex += 'static const char * IndexToPhysDevFeatureString[] = {\n'
```
Comment (mikes-lunarg) [inline scripts/helper_file_generator.py]: You could scope IndexToPhysDevFeatureString inside GetPhysDevFeatureString

File: layers/core_validation.cpp (hunk 1 above)
Comment (mark-lunarg) [inline layers/core_validation.cpp]: Ta-da.

File: scripts/helper_file_generator.py (hunk 2 above)
Comment (mark-lunarg) [inline scripts/helper_file_generator.py]: Yes, nicer.

**Pull request merged**